From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Wed, 3 Jan 2024 17:05:54 +0000
Subject: [PATCH] config: add new needed deps to fetch filter lists

This is needed to generate subresource filtering component used in browser
for adblocking.
---
 DEPS | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/DEPS b/DEPS
index 55e524cfb710e..744f2f413adc5 100644
--- a/DEPS
+++ b/DEPS
@@ -4798,6 +4798,17 @@ hooks = [
                '--input-public',
                'src/chromeos/tast_control_disabled_tests_public_builders.txt'],
   },
+  {
+    'name': 'fetch_filter_lists',
+    'condition': 'checkout_android',
+    'action': ['python3',
+              'src/vanadium/android_config/filter_lists/filter_list_download.py',
+              '--output',
+              'src/vanadium/android_config/filter_lists/filter_lists_easylist.txt',
+              '--urls',
+              'https://easylist.to/easylist/easylist.txt',
+              'https://easylist.to/easylist/easyprivacy.txt']
+  },
 
 ]
 
